libs/features/mas/src/variants/plans.js

```diff
@@ -9,8 +9,10 @@ export const PLANS_AEM_FRAGMENT_MAPPING = {
   description: { tag: 'div', slot: 'body-xs' },
   mnemonics: { size: 'l' },
   callout: {tag: 'div', slot: 'callout-content'},
+  quantitySelect: { tag: 'div', slot: 'quantity-select' },
   stockOffer: true,
   secureLabel: true,
+  badge: true,
```

Comment:
why this is needed?

Reply:
Card editor reads this mapping to decide which authoring fields will be displayed for 'plans' cards. For card badge (the yellow thing in the upper right corner) we don't have slots but it needs to be mentioned here to display that Badge text field in card editor. The same as for stock checkbox.